.github/workflows/main.yml

```diff
@@ -0,0 +1,24 @@
+name: Github Actions
+
+on: [push, pull_request]
+
+jobs:
+  host-x86:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        compiler: [gcc-12, clang]
```

Comment:
After bumping Ubuntu to 24.04, the version of gcc becomes 13. I don't think you have to install other gcc versions than the default.

Reply:
Should I still specify gcc version or just remove it?

Reply:
If there is no reason to specify the versioning, let's minimize the actions.